### DISCUSSION TITLE
Should it be an error for letter case to differ between file path on disk and in an identifier?

### DISCUSSION THREAD

**Discussion by kendalldita** [dita-ot/dita-ot#4348] [Q&A]
I mean DOTJ083E. I can't believe this is the first time I've seen this. If a user gets the case wrong in a filepath it's an error (in dita-ot 3.6.1)

**Comment by chrispy-snps** [comment]
@kendalldita - I don't know the details of this message. Are you running on Windows? If so, perhaps this is a guard check to insure that content is not created that can't be published in case-sensitive filesystems such as Linux.

**Comment by kendalldita** [comment]
Yes windows. The code (in 3.6.1) checks if the requested path and the actual  path on the filesystem are Object.equals. So, if someone types:

c:\docs

the paths aren't equal on windows, but they are if they had typed:

C:\docs

in the cases I saw.

So, I don't think that would affect non-windows operating systems (except vms?), sine all the files will likely have their same case preference. It would be nice to have an option to disable the equality test.
